camelot/admin/action/list_filter.py

```diff
@@ -80,6 +80,9 @@ def __init__(self, attribute, default=All, verbose_name=None):
         self.attributes = None
         self.filter_names = []
 
+    def get_name(self):

```

Comment:
denk dat we dan in de __init__ een assert moeten steken dat attribute een string is.

op zich zou het nuttig zijn om niet strings in die attribute toe te laten, maar dan moeten we wel rekening houden met die name

Reply:
Moet ik dan alle attribute argumenten veranderen in strings?

In de TypeComboBoxFilter is er ook deze assert: `assert isinstance(attribute.type, IntEnum)`
Ik denk niet dat deze klasse zou werken met een string.

De string representatie lijkt meestel wel aanvaardbaar voor die get_name: 
`<class 'sqlalchemy.orm.attributes.InstrumentedAttribute'>` geeft bijvoorbeeld `FinancialSecurityOrderLine.described_by`

Reply:
die string representatie lijkt me idd ok te zijn